src/main/java/com/dongtronic/diabot/commands/nightscout/NightscoutCommand.kt

```diff
@@ -418,4 +422,19 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand(category, nu
 
         return null
     }
+
+    private fun getDisplayOptions(user: User): String {
```

Comment:
Why not use an array? They're easier to search through.

Reply:
Converted to an array in the newest commits